src/nvim/api/ui.c

```diff
@@ -631,7 +631,7 @@ static void remote_ui_raw_line(UI *ui, Integer grid, Integer row, Integer startc
       remote_ui_highlight_set(ui, (int)clearattr);
       // legacy eol_clear was only ever used with cleared attributes
       // so be on the safe side
-      if (clearattr == 0 && clearcol == Columns) {
+      if (clearattr == 0 && clearcol == g_columns) {
```

Comment:
I strongly favor `g_` for globals but this will cause massive churn. Waiting for comments from other core contributors.

Reply:
Why not `COLUMNS`?

Reply:
We only use all caps for macros. Constants are "k" prefixed. Those rules apply regardless of scope.

Globals are something else. `g_` tells me something is clearly a global. Less important with advanced tooling these days, but I think there's always a benefit to having unambiguous lexical signals that don't depend on tools.

Reply:
Ah right. I confused "globals" with "constants".

Reply:
> I strongly favor g_ for globals but this will cause massive churn. Waiting for comments from other core contributors.

It started off relatively okay, but they're used everywhere which results in this many changed files..